### DISCUSSION TITLE
Residual Error?

### DISCUSSION THREAD

**Discussion by z-a-kane** [nlmixrdevelopment/nlmixr#484] [Q&A]
Hi, I was just wondering is it possible to use two residual error terms in nlmixr? I am trying to transfer a NONMEM model to nlmixr and in NONMEM I was using:
$ERROR
CP   = A(2) / V2
;------ Statistical model
IF(CMT==2)   IPRED = CP
IF(CHLD==1) Y = IPRED * (1 + EPS(1))  ; EPS(1) is proportional residual error on pead data
IF(CHLD==0) Y = IPRED * (1 + EPS(2))  ; EPS(2) is proportional residual error on adult  data

Many thanks
Zoe

**Comment by mattfidler** [comment]
You would treat it as a multiple endpoint model;

https://nlmixrdevelopment.github.io/nlmixr/articles/multiple-endpoints.html

The first endpoint would be children and the second would be pediatrics.  You could change `DVID` or `CMT` to predict adult/ped data.

**Reply by z-a-kane** [reply]
Thank you.
cheers
Zoe
